noir-projects/aztec-nr/aztec/src/context/avm_context.nr

```diff
@@ -200,9 +197,12 @@ impl Empty for AvmContext {
 
 // Helper functions
 fn gas_for_call(user_gas: GasOpts) -> [Field; 2] {
+    // It's ok to use the max possible gas here, because the gas will be
+    // capped by the gas left in the (STATIC)CALL instruction.
+    let MAX_POSSIBLE_FIELD: Field = 0 - 1;
     [
-        user_gas.l2_gas.unwrap_or_else(|| l2_gas_left() - NESTED_CALL_L2_GAS_BUFFER),
-        user_gas.da_gas.unwrap_or_else(|| da_gas_left())
+        user_gas.l2_gas.unwrap_or_else(|| MAX_POSSIBLE_FIELD),
+        user_gas.da_gas.unwrap_or_else(|| MAX_POSSIBLE_FIELD)
```

Comment:
```suggestion
        user_gas.l2_gas.unwrap_or(MAX_POSSIBLE_FIELD),
        user_gas.da_gas.unwrap_or(MAX_POSSIBLE_FIELD)
```

Just because I like leaving at least a comment on a PR when reviewing :-P

Reply:
Haha, thanks! I wanted that actually :)